test/unit/wiki_extensions_child_pages_count_test.rb

```diff
@@ -0,0 +1,22 @@
+require File.dirname(__FILE__) + '/../test_helper'
+
+class WikiExtensionsChildPagesCountTest < Redmine::HelperTest
+  include ApplicationHelper
+  include ERB::Util
+
+  fixtures :projects, :wikis, :wiki_pages, :wiki_contents, :issues
+
+  def test_child_pages_count
+    wiki_page = WikiPage.find(2)
+
+    assert_equal 3, wiki_page.descendants.size
```

Comment:
Is this needed?

Reply:
I thought it was useful for developers to read this test because they cannot know what number 3 is.
But I think it's not necessary here. I will remove it because they can know it from the implementation of child_pages macro.

Reply:
fix: [b7f9d70](https://github.com/otegami/redmine_wiki_extensions/pull/1/commits/b7f9d7059988ff1b27066caec1f2dc3f493776b2)